README.md

```diff
@@ -54,10 +54,10 @@ In case following error occurs, it means there is a test module `XYZ` that canno
 
 ```txt
 File "setup.py", line 499, in loadTestsFromModule
-    testsuites = ScanningLoader.loadTestsFromModule(self, module)
+testsuites = ScanningLoader.loadTestsFromModule(self, module)
 File "build/bdist.linux-x86_64/egg/setuptools/command/test.py", line 37, in loadTestsFromModule
 File "/usr/lib64/python2.7/unittest/loader.py", line 100, in loadTestsFromName
-    parent, obj = obj, getattr(obj, part)
+parent, obj = obj, getattr(obj, part)
```

Comment:
because it's not valid rst otherwise, which is what this pr is supposed to check for